documentation/endpoints/posts.md

```diff
@@ -40,29 +40,21 @@ Example Response:
 ```
 {
   "data": {
-    "signup_id": 31,
```

Comment:
So, this might not be needed for merge since we don't really know what people need, but I feel like we still want to return the signup, northstar id, campaign id, etc that the post is related to.